ui/src/app/utils/simpleSortByKey.ts

```diff
@@ -2,12 +2,16 @@
  * Simple sort objects by key value.
  *
  * @param key - the key of the objects to sort by
+ * @param reverse - whether to reverse the results
  * @returns sort function
  */
 export const simpleSortByKey =
-  <O, K extends keyof O>(attr: K): ((a: O, b: O) => number) =>
+  <O, K extends keyof O>(
+    key: K,
+    { reverse }: { reverse: boolean } = { reverse: false }
```

Comment:
I don't think this needs the `{ reverse: boolean }` type as it will get inferred from the default value.